vizro-core/examples/features/app.py

```diff
@@ -404,6 +411,9 @@
             column="size",
             selector=vm.Slider(title="Slider (Tips - size)", step=1, value=2),
         ),
+        vm.Filter(
+            targets=["table-gapminder"], column="year", selector=vm.DatePicker(title="Date Picker (Gapminder - year)")
```

Comment:
The year column is actually not a very nice example for this. I like your dev example more for example. What I would suggest is you use the `px.data.stocks()`, create a line chart and add it to this page. And this Filter will then take the date column and filter on that data set. Then just re-name to "Date Picker (Stocks - date)"

Reply:
Don't forget to create the yaml example out of that as well then :)